docs/en/latest/admin-api.md

```diff
@@ -784,6 +784,8 @@ Return response from etcd currently.
 | key         | True     | Private key             | https private key                                                                                                                                          |                                                  |
 | certs       | False    | An array of certificate | when you need to configure multiple certificate for the same domain, you can pass extra https certificates (excluding the one given as cert) in this field |                                                  |
 | keys        | False    | An array of private key | https private keys. The keys should be paired with certs above                                                                                             |                                                  |
+| client.ca | False | Certificate| set the CA certificate which will use to verify client. This feature requires OpenResty 1.19+.                                                            |                                                  |
+| client.depth | False | Certificate| set the verification depth in the client certificates chain, default to 1. This feature requires OpenResty 1.19+.                                                            |                                                  |
```

Comment:
The first letter should be capitalized.  set --> Set
```
set the CA certificate which will use to verify client.
```

Reply:
None of the description in this table start with uppercase letter.

Reply:
Got it.